### ISSUE TITLE
[RFE] Filter requires per subpackage

### ISSUE THREAD

**Issue by voxik** [rpm-software-management/rpm#2720]
Unless I am mistaken, the whole family of `%__requires_exclude` works globally. However, it would be nice if the excludes were more fine grained.

For example, I have sub-package, where I'd like to replace the autogenerated `Requires: rubygem(rdoc)` by soft dependency `Recommends: rubygem(rdoc)`. However, at the moment, I can exclude the `rubygem(rdoc)` only globally. Adding it back into appropriate places would be painful.

**Comment by ffesti** [comment]
Yeah, giving the packager more control here is something we generally want. The dependency generators are a bit clunky as soon as they don't exactly what one wants/needs.
